docs/root/faq/edge.rst

```diff
@@ -0,0 +1,85 @@
+.. _faq_edge:
+
+Can I use Envoy as an edge proxy?
+=================================
+
+Envoy is a production-ready edge proxy, and performs well in that role, however,
+the default settings are tailored for the service mesh use case, and some values
+need to be adjusted when using Envoy as an edge proxy.
+
+TCP proxies should configure: :ref:`overload_manager <config_overload_manager>`,
+:ref:`listener buffer limits <envoy_api_field_Listener.per_connection_buffer_limit_bytes>` to 32,768 bytes
+and :ref:`cluster buffer limits <envoy_api_field_Cluster.per_connection_buffer_limit_bytes>` to 32,768 bytes.
+
+HTTP proxies should additionally configure:
+:ref:`use_remote_address <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.use_remote_address>` to true,
```

Comment:
should we include some kind of explanation for why these config options are recommended for edge deployments?

Reply:
+1; if we can explain the reasoning, operators will be better positioned to make informed changes.

Reply:
As part of this, it would also be good to address my question here: #8412 

Since the config option is related, and this question is specifically for an edge proxy deployment

Reply:
@snowp added.